packages/forms/src/Components/Concerns/CanBeMarkedAsRequired.php

```diff
@@ -15,8 +15,8 @@ public function markAsRequired(bool | Closure $condition = true): static
         return $this;
     }
 
-    public function isMarkedAsRequired(): string
```

Comment:
how tf did this one slip in...

Reply:
Lol, that's what I wondered too. Apparently I am to blame. 😅😂